### ISSUE TITLE
No discernible inference after ~360000 iterations on 8k audio.

### ISSUE THREAD

**Issue by ziafkhan** [NVIDIA/waveglow#158]
Here is the config.json i have adapted for 8k files. The learning rate was reduced after nearly 300000 iterations.
```{
    "train_config": {
        "fp16_run": false,
        "output_directory": "checkpoints",
        "epochs": 100000,
        "learning_rate": 1e-5,
        "sigma": 1.0,
        "iters_per_checkpoint": 1000,
        "batch_size": 4,
        "seed": 1234,
        "checkpoint_path": "checkpoints/waveglow_316000",
        "with_tensorboard": true
    },
    "data_config": {
        "training_files": "train_files.txt",
        "segment_length": 16000,
        "sampling_rate": 8000,
        "filter_length": 512,
        "hop_length": 100,
        "win_length": 512,
        "mel_fmin": 0.0,
        "mel_fmax": 2232.0
    },
    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://localhost:54321"
    },

    "waveglow_config": {
        "n_mel_channels": 80,
        "n_flows": 12,
        "n_group": 8,
        "n_early_every": 4,
        "n_early_size": 2,
        "WN_config": {
            "n_layers": 8,
            "n_channels": 128,
            "kernel_size": 3
        }
    }
}
```
I had to reduce the n_channels to get a decent batch size.
I am training on a little over 18 hrs of single speaker data.
I also trained the tacotron2 with similar data_config and it seems to have trained well. I checked using a combination of (trained tacotron2 + downloaded waveglow model) and heard legible sentences.
Currently the wav outputs have no meaningful sentences at all.
Any help will be much appreciated.

**Comment by xDuck** [comment]
Have you made any progress with this? I am also attempting to train an 8k model

**Comment by ziafkhan** [comment]
This was the latest version of config i tried. I also increased the n_channels back to 256. Though I could now make out the voice as that of the speaker, she was unfortunately still speaking gibberish. Sorry. :(
```
"data_config": {
        "training_files": "train_files_8k.txt",
        "segment_length": 16000,
        "sampling_rate": 8000,
        "filter_length": 1024,
        "hop_length": 64,
        "win_length": 1024,
        "mel_fmin": 0.0,
        "mel_fmax": 5680.0
    }
```
I have tried filter_lengths of 384 and 512 as well. 
In the end I tested the default parameters in config.json with audio upsampled to 22050 Hz and though noisy, the output was legible. So it does not seem to me to be a data issue either.

**Comment by rafaelvalle** [comment]
https://github.com/NVIDIA/waveglow/issues/168#issuecomment-560695781

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
